src/luarocks/loader.lua

```diff
@@ -22,27 +22,9 @@ local manif_core = require("luarocks.manif_core")
 local deps = require("luarocks.deps")
 local util = require("luarocks.util")
 
--- Workaround for wrappers produced by older versions of LuaRocks
```

Comment:
I think such things should be described in the commit message and possibly even be moved into separate commit.